src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int CIRCLE_RADIUS = 10;
+    public static final int MAX_OF_BOUND = 40;
+    public static final String COLOR = "white";
+    private Random random = new Random();
+    private final Figure defaultFigure = new Circle(CIRCLE_RADIUS);
+
+    public Figure getRandomFigure() {
+        Figure[] figures = new Figure[]{
+                new Circle(random.nextInt(MAX_OF_BOUND)),
+                new IsoscelesTrapezoid(random.nextInt(MAX_OF_BOUND), random.nextInt(MAX_OF_BOUND),
+                        random.nextInt(MAX_OF_BOUND)),
+                new Rectangle(random.nextInt(MAX_OF_BOUND), random.nextInt(MAX_OF_BOUND)),
+                new RightTriangle(random.nextInt(MAX_OF_BOUND), random.nextInt(MAX_OF_BOUND)),
+                new Square(random.nextInt(MAX_OF_BOUND))};
+        return figures[random.nextInt(figures.length)];
```

Comment:
If random gives out the same number, we will have the same figures.
It is better to get a number, make a switch on the number and create a new figure each time, if it goes into the number.
It is also better to create shapes in a separate method.